frontend/src/api/Search.js

```diff
@@ -1,137 +1,151 @@
 import axios from 'axios';
```

Comment:
Is this needed or a convenience?  Why not use just the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)?